app/models/wizard/component.rb

```diff
@@ -2,7 +2,9 @@ module Wizard
   class Component
     include ActiveModel::Model
     include MongoMapper::EmbeddedDocument
-
+    
+    one :layout, class: PrintLayout::Layout
```

Comment:
components are currently layout agnostic, they just know how to draw themselves but not how to layout.